readme.md

```
@@ -1 +1,46 @@
+
+### Step 2: Setup Environment Variables
+Create a .env file in the root of your project and add the DATABASE_URL for your CockroachDB database connection.
+```
+DATABASE_URL=postgresql://user:password@localhost:26257/database_name?sslmode=disable
+```
+Replace user, password and database_name with your actual database credentials.
+
+### Step3: Configure Flyway (Optional, for Migrations)
+If you want to use Flyway for managing migrations, ensure your flyway.conf is correctly configured with database connection strings. 
+Example `flyway.conf`
+```
```

Markdownlint

Fenced code blocks should have a language specified

(MD040, fenced-code-language)